docs/index.md

```diff
@@ -33,8 +33,8 @@ A fully fledged audio module created for music apps. Provides audio playback, ex
 
 ## Example
 
-If you want to get started with this module, check the [Getting Started](./API.md) page.
-If you want detailed information about the API, check the [Documentation](./Documentation.md).
+If you want to get started with this module, check the [Getting Started](https://react-native-kit.github.io/react-native-track-player/api/) page.
+If you want detailed information about the API, check the [Documentation](https://react-native-kit.github.io/react-native-track-player/documentation/).
```

Comment:
Is there a reason to get back into using full URLs here?

Reply:
Happy to change it – but we are also using full links in the index at the top of README – should I change those into relative links also?

Reply:
Do you mean in the `README.md`? Unfortunately relative links don't work as expected there, but we should use relative links in every `docs/` page where possible